resources/table/table.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+
+interface TableProps<T> {
```

Comment:
Thought we were using tailwind css components

Reply:
Actually TailwindCSS only provides us with the CSS components, the react components by Tailwind are paid.

Reply:
Sad 
Can search for others